pkgs/tools/system/kanata/default.nix

```diff
@@ -0,0 +1,35 @@
+{ fetchFromGitHub
+, lib
+, libevdev
+, pkg-config
+, rustPlatform
+, withCmd ? false
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "kanata";
+  version = "1.0.5";
+
+  src = fetchFromGitHub {
+    owner = "jtroo";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-sL9hP+222i8y0sK3ZEx66yXBTgZp5ewoPUlZS4XnphY=";
+  };
+
+  cargoHash = "sha256-uhN1UdwtU0C0/lpxUYoCcMLABFTPNO5wKsIGOBnFpzw=";
+
+  buildFeatures = lib.lists.optional withCmd "cmd";
```

Comment:
```suggestion
  buildFeatures = lib.optional withCmd "cmd";
```

Reply:
I prefer `lib.lists.optional` for clarity